modules/sdk-core/src/bitgo/bitgoBase.ts

```diff
@@ -0,0 +1,43 @@
+import * as superagent from 'superagent';
+import { EnvironmentName } from './environments';
+
+export interface BitGoRequest<ResultType = any> extends superagent.SuperAgentRequest {
+  result: (optionalField?: string) => Promise<ResultType>;
+}
+
+export interface DecryptOptions {
+  input: string;
+  password?: string;
+}
+
+export interface EncryptOptions {
+  input: string;
+  password?: string;
+}
+
+export interface GetSharingKeyOptions {
+  email: string;
+}
+
+export interface IRequestTracer {
+  inc(): void;
+  toString(): string;
+}
+
+export interface BitGoBase {
```

Comment:
`BitGoAPI` in `@bitgo/sdk-api` needs to implement this interface.

Reply:
Should it be part of ticket BG-48170 or should we use a different ticket for this?

Reply:
it should be part of this